myriad-executor/src/main/java/org/apache/myriad/executor/MyriadExecutor.java

```diff
@@ -70,10 +70,11 @@ public void launchTask(final ExecutorDriver driver, final TaskInfo task) {
 
   @Override
   public void killTask(ExecutorDriver driver, TaskID taskId) {
-    LOGGER.debug("killTask received for taskId: " + taskId.getValue());
+    String taskIdString = taskId.getValue();
+    LOGGER.debug("killTask received for taskId: " + taskIdString);
     TaskStatus status;
 
-    if (!taskId.toString().contains(MyriadExecutorAuxService.YARN_CONTAINER_TASK_ID_PREFIX)) {
+    if (!taskIdString.contains(MyriadExecutorAuxService.YARN_CONTAINER_TASK_ID_PREFIX)) {
```

Comment:
I've got no opinion on getValue vs toString, the main issue was I uses taskId.toString more than once so thought I should keep it around.  I'll set taskIdString = taskId.toString()